src/smt.h

```diff
@@ -48,19 +46,34 @@ sort bvSort(unsigned bw);
 sort boolSort();
 sort arraySort(const sort &domain, const sort &range);
 
+class Expr;
+using ExprVec = std::vector<Expr>;
```

Comment:
Let's remove this and use std::vector<Expr>. :)

Reply:
Fixed